lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/DriverRuntimeRestartConfiguration.java

```diff
@@ -33,6 +39,13 @@
   private DriverRuntimeRestartConfiguration() {
   }
 
-  // TODO: bind service handlers in REEF-483
-  public static final ConfigurationModule CONF = new DriverRuntimeRestartConfiguration().build();
+  public static final ConfigurationModule CONF = new DriverRuntimeRestartConfiguration()
+
+      // TODO: decide on whether to call the restart functions on DriverRestartManager directly or to use this class.
```

Comment:
Please don't leave commented out code in. If you add a TODO, add it in the form `// TODO[REEF-xxx]: Message`